lib/src/flutter_mapbox_navigation.dart

```diff
@@ -66,10 +65,14 @@ class MapBoxNavigation {
   /// [options] options used to generate the route and used while navigating
   /// Begins to generate Route Progress
   ///
-  Future<bool?> startNavigation(
-      {required List<WayPoint> wayPoints, MapBoxOptions? options}) async {
+  Future<bool?> startNavigation({
```

Comment:
This function should be the only thing that changes in this file. Please remove all formatting changes you are pushing through.

Reply:
All the formatting changes are because of the default flutter formatter rules like new line after 80 characters, spacing imports to split packages imports from relatives imports and so on.

I'll suggest to stick with "official" rules or it will be impossible for anyone contribute to this package.

Reply:
so the formatting changes should be submitted in a separate PR to clear separation